activemq/CVE-2015-5254/README.md

```diff
@@ -28,7 +28,7 @@ The exploit process is as follows:
 2. send payload to port 61616
 3. Access the web management page and read the serialization messages, then u can trigger vulnerability.
 
-To exploit this environment we will use [jmet](https://github.com/matthiaskaiser/jmet) (Java Message Exploitation Tool). First download the jar file of jmet, and create an external folder in the same directory (otherwise it may occur the error that the folder does not exist).
+To exploit this environment we will use [jmet](https://github.com/matthiaskaiser/jmet) (Java Message Exploitation Tool). First download the jar file of jmet, and create folder called __`external`__ in the same directory (otherwise it may occur the error that the folder does not exist).
```

Comment:
Just like https://github.com/vulhub/vulhub/pull/174, it's better to use asterisks, thanks.